config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+
+  resources :drivers
+  resources :passengers
+  resources :trips
+  get('passengers/:id/trips', to:'trips#create', as: 'passenger_trip')
+  post('passengers/:id/trips', to: 'trips#create')
```

Comment:
I don't think the `get` route is set up correctly here, nor is it used by your application.

Both it and the `post` route could be nested under `resoruces :passengers`:

```ruby
resources :passengers do
  resources :trips, only: [:index, :create]
end
```